openedx/core/djangoapps/courseware_api/views.py

```diff
@@ -98,7 +98,7 @@ def __init__(self, course_key, request, username=''):
             course_key=course_key,
             is_global_staff=self.original_user_is_global_staff,
         )
-        self.can_view_certificate = certificates_viewable_for_course(self.course)
+        self.can_view_certificate = certificates_viewable_for_course(self.enrollment_object.course_overview)
```

Comment:
Oh, interesting, so we already had access to the course overview. Nice! I think that should be good. I was expecting us to have to make a call to the `get_course_overview_or_none` function. Now we don't have to make _any_ extra calls 👍